public/app/features/explore/state/utils.ts

```diff
@@ -34,6 +35,21 @@ const loadGraphStyle = (): ExploreGraphStyle => {
   return toGraphStyle(data);
 };
 
+const LOGS_VOLUME_ENABLED_KEY = SETTINGS_KEYS.enableVolumeHistogram;
+export const storeLogsVolumeEnabled = (enabled: boolean): void => {
+  store.set(LOGS_VOLUME_ENABLED_KEY, enabled ? 'true' : 'false');
+};
+
+const loadLogsVolumeEnabled = (): boolean => {
+  const data = store.get(LOGS_VOLUME_ENABLED_KEY);
+  // we default to `enabled=true`
+  if (data === 'false') {
+    return false;
+  } else {
+    return true;
+  }
```

Comment:
hmmm... i see your point (i think we could even go with `return enabled !== 'false'`... i did this because i really wanted to emphasize that the only case where it is `false`, is when the value is the string `"false"`.

hmm.. i changed it to:
```ts
  if (data === 'false') {
    return false;
  }

  return true;
```

is that better?

Reply:
yup, it was just a nit, actually your solution reads way better 👍